obs-studio-server/source/gs-vertexbuffer.cpp

```diff
@@ -102,10 +102,13 @@ GS::VertexBuffer::VertexBuffer(uint32_t maximumVertices)
 
 	// Allocate GPU
 	obs_enter_graphics();
-	m_vertexbuffer = gs_vertexbuffer_create(m_vertexbufferdata, GS_DYNAMIC);
-	std::memset(m_vertexbufferdata, 0, sizeof(gs_vb_data));
 	m_vertexbufferdata->num     = m_capacity;
 	m_vertexbufferdata->num_tex = m_layers;
+	m_vertexbuffer              = gs_vertexbuffer_create(m_vertexbufferdata, GS_DYNAMIC);
+	// No need to set all the data to zero - the buffer data should be valid
+	// otherwise, during device rebuild, the d3d11 couldn't recreate vertexbuffer
+	//std::memset(m_vertexbufferdata, 0, sizeof(gs_vb_data));
```

Comment:
Why only commenting and not removing then?

Reply:
I added comments for visibility. Will remove them